registry/tests/admin_test.py

```diff
@@ -59,3 +59,85 @@ def testListPackagesNotAdmin(self):
         )
         assert resp.status_code == requests.codes.forbidden
 
+    def testAuditUser(self):
+        # actions as user test_user
+        self._mock_admin()
+        resp = self.app.get(
+            '/api/audit/{usr}/'.format(
+                usr=self.user
+            ),
+            headers={
+                'Authorization': self.user
+            }
+        )
+        data = json.loads(resp.get_data()).get('events')
+        assert(len(data) == 3)
```

Comment:
`assert` is an operator, not a function; use a space and no parens.

It can actually cause very subtle bugs:
```
assert False, "This fails"
assert(False, "This does NOT fail")
```